docs/BRIDGE_DESIGN.md

```
@@ -0,0 +1,181 @@
+        C
+    end
+
+    subgraph "Destination"
+        D
+        E
+    end
+```
+
+### ASCII Architecture
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)